cmake/CMakeLists.txt

```diff
@@ -161,10 +161,6 @@ if(onnxruntime_USE_VALGRIND AND NOT WIN32)
   add_definitions(-DRE2_ON_VALGRIND=1)
 endif()
 
-if(WIN32)
```

Comment:
the '/W' flag also gets set here:
https://github.com/microsoft/onnxruntime/blob/8e493ee3aa9cff9261ae970c745dcb8960f23cc7/cmake/CMakeLists.txt#L971-L975

Reply:
But if you remove it, the third-party libraries won't be compiled with warning level 3. The default value is 1.

Reply:
found another way to set it.